src/notification-hub/azext_notification_hub/custom.py

```diff
@@ -0,0 +1,360 @@
+                             namespace_name,
+                             notification_hub_name):
+    return client.get(resource_group_name=resource_group_name, namespace_name=namespace_name, notification_hub_name=notification_hub_name)
+
+
+def list_notificationhubs_hub(cmd, client,
+                              resource_group_name,
+                              namespace_name):
+    return client.list(resource_group_name=resource_group_name, namespace_name=namespace_name)
+
+
+def check_notification_hub_availability_notificationhubs_hub(cmd, client,
+                                                             resource_group_name,
+                                                             namespace_name,
+                                                             notification_hub_name):
+    body = {"name": notification_hub_name}
+    return client.check_notification_hub_availability(resource_group_name=resource_group_name, namespace_name=namespace_name, parameters=body)
+
+
+def regenerate_keys_notificationhubs_hub(cmd, client,
+                                         resource_group_name,
+                                         namespace_name,
+                                         notification_hub_name,
+                                         rule_name,
+                                         policy_key):
+    return client.regenerate_keys(resource_group_name=resource_group_name, namespace_name=namespace_name, notification_hub_name=notification_hub_name, authorization_rule_name=rule_name, policy_key=policy_key)
+
+
+def get_pns_credentials_notificationhubs_hub(cmd, client,
+                                             resource_group_name,
+                                             namespace_name,
+                                             notification_hub_name):
+    return client.get_pns_credentials(resource_group_name=resource_group_name, namespace_name=namespace_name, notification_hub_name=notification_hub_name)
+
+
+def list_keys_notificationhubs_hub(cmd, client,
+                                   resource_group_name,
+                                   namespace_name,
+                                   notification_hub_name,
+                                   rule_name):
+    return client.list_keys(resource_group_name=resource_group_name, namespace_name=namespace_name, notification_hub_name=notification_hub_name, authorization_rule_name=rule_name)
+
+
+def debug_send_notificationhubs_hub(cmd, client,
+                                    resource_group_name,
+                                    namespace_name,
+                                    notification_hub_name,
+                                    notification_format,
+                                    message=None,
+                                    title='',
+                                    payload=None,
+                                    tag=None):
+
+    if message is not None:
+        if notification_format == 'gcm':
```

Comment:
just one suggestion. Could you paste a link where you learn the knowledge from. In case in the future, someone else can help you maintain this extension.